mPower2/mPower2/UIColor+mPowerTheme.swift

```diff
@@ -42,6 +42,10 @@ extension UIColor {
         return UIColor.royal500
     }
 
+    @objc open class var veryLightPrimaryTintColor: UIColor {
+        return UIColor.royal400
+    }
+
```

Comment:
Why is this necessary? It's the same color as `lightPrimaryTintColor` defined below.